### ISSUE TITLE
Support ollama keep_alive parameter in plugin settings

### ISSUE THREAD

**Issue by dorianbrown** [logancyang/obsidian-copilot#423]
**Is your feature request related to a problem? Please describe.**

Currently by default ollama keeps the model in memory until 5 minutes has passed. This consumes RAM/VRAM, and can impact the performance of other programs being run after using obsidian-copilot. It would be nice if we could configure the timeout to be something shorter (or longer if wanted). 

**Describe the solution you'd like**

Provide a `keep_alive` option in the ollama configuration section of the plugin.

**Comment by Tycho-S** [comment]
Yes this would really be great to have. I personally prefer it to keep the models in memory altogether so there is no loading time.

Another solution might be to remove the keep_alive parameter from the API call altogether. Then it will just obey whatever the user has set as keepalive on the server. So the user is able to control it there then, no need to create a setting for it in copilot.

**Comment by logancyang** [comment]
Thanks for reporting this, and sorry for the slow reply.

Copilot v3 is in code freeze, so we're closing this as not planned. v4 is a ground-up rewrite and is launching soon, with a very different architecture around agent mode.

If this is still an issue for you in v4 once it lands, please open a new issue and we'll take a fresh look. 🙏
